gcp/api/server.py

```diff
@@ -967,66 +1008,51 @@ def _query_by_generic_version(
     version: str,
 ):
   """Query by generic version."""
```

Comment:
Please improve this docstring while you're here: https://google.github.io/styleguide/pyguide.html#s3.8.3-functions-and-methods